react-native.config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+'use strict';
+
+const {
+  commands: commandsIOS,
+  ...ios
+} = require('@react-native-community/cli-platform-ios');
+const {
+  commands: commandsAndroid,
+  ...android
+} = require('@react-native-community/cli-platform-android');
+
+module.exports = {
+  reactNativePath: '.',
+  commands: [...commandsIOS, ...commandsAndroid],
+  platforms: {ios, android},
```

Comment:
maybe let's be explicit about what we put here, because:
1) it will serve as a documentation
2) if we ever export some extra helpers from these modules, they won't break the configuration

so, even though more verbose, I'd expect:
```js
platforms: {
  ios: {
    linkConfig: ios.linkConfig,
    projectConfig: ios.projectConfig,
    dependencyConfig: ios.dependencyConfig
  },
  android: {
    linkConfig: android.linkConfig,
    projectConfig: android.projectConfig,
    dependencyConfig: android.dependencyConfig
  }
}
```

Reply:
Yeah, good idea! Let me do that in a second.